e2e-tests/accessibility.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { test, expect } from "@playwright/test";
+import AxeBuilder from "@axe-core/playwright";
+import { createHtmlReport } from "axe-html-reporter";
+import fs from "fs";
+import { login } from "./helpers";
```

Comment:
Usually all quotes should be single. Could you please check if linting is working as expected for the `e2e-tests` directory?